code/3_data_processing.py

```diff
@@ -45,67 +45,84 @@
 
 import os
 import sys
+
     
 def main():
-  
-    from pyspark.sql import SparkSession
-    from pyspark.sql import functions as sqlfn
-
-    spark = (
-        SparkSession.builder.appName("Airline Data Exploration")
-        .config("spark.executor.memory", "8g")
-        .config("spark.executor.cores", "4")
-        .config("spark.driver.memory", "20g")
-        .config("spark.executor.instances", "4")
-        .config("spark.yarn.access.hadoopFileSystems", os.environ["STORAGE"])
-        .getOrCreate()
-    )
+    import cml.data_v1 as cmldata
+    from pyspark import SparkContext
+    from pyspark.sql import functions as sqlfn   
+    
+    ### cleanup needed start
+    # These shound't be set after cml.data adds iceberg support
+    import glob
+
+    jars = glob.glob("/opt/spark/optional-lib/*.jar")
+    jarsList = ",".join(jars)
+    SparkContext.setSystemProperty("spark.jars",jarsList)
```

Comment:
ping alex for the better snippet, we found an issue with applying all jars, so now we are careful in only applying iceberg and hwc jar if they exist.

Reply:
Ok, I copied the code from the `cml.data` library.

Reply:
👍